packages/debugger/src/panels/sources/body.ts

```diff
@@ -123,6 +123,15 @@ export class SourcesBody extends Widget {
 
     requestAnimationFrame(() => {
       EditorHandler.showCurrentLine(this._editor.editor, frame.line);
+      try {
+        // Scroll to the highlighted line.
+        this.node
+          .getElementsByClassName(LINE_HIGHLIGHT_CLASS)
+          .item(0)
+          ?.scrollIntoView();
+      } catch {
+        // no op
+      }
     });
```

Comment:
Should `EditorHandler.showCurrentLine` have an optional (default?) scroll argument instead and then emit `EditorView.scrollIntoView` effect so that CodeMirror handles it? I worry that we are doing something which depend on implementation detail of CodeMirror DOM here and will break in a future update (especially since there is no test).